python/lib/session.py

```diff
@@ -66,10 +75,34 @@ def create_session(self, db):
             print("Creating visit " + self.visit_label \
                   + " for CandID "  + self.cand_id)
 
+        column_names = ('CandID', 'Visit_label', 'CenterID')
+        values = (self.cand_id, self.visit_label, str(self.center_id))
+
+        if self.project_id:
+            column_names = column_names + ('ProjectID',)
+            values = values + (str(self.project_id),)
+
+        if self.subproject_id:
+            column_names = column_names + ('SubprojectID',)
+            values = values + (str(self.subproject_id),)
+        
+        current_stage = None
+        current_stage_info = db.pselect(
+            "SELECT Value FROM Config WHERE ConfigID = (SELECT ID FROM ConfigSettings WHERE Name='default_bids_stage')",
+            []
+        )
+
+        if(len(current_stage_info) > 0):
+            current_stage = current_stage_info[0]['Value']
+
+        if current_stage:
+            column_names = column_names + ('Current_stage',)
+            values = values + (current_stage,)
+            
```

Comment:
beyond setting the session table field, should this trigger the battery population in Loris, @cmadjar ? pros/cons

Reply:
that should have the exact same behaviour as the Perl insertion pipeline, meaning setting the stage to "Not Started" and not populating the battery. That should be done by the data entry side of LORIS.